model/inp/ww3_grid.inp

```diff
@@ -423,6 +423,39 @@ $                                 provides 3 partitions of both x & y component,
 $                                 with a reasonable accuracy for using in
 $                                 a climate model.
 $
+$ Langmuir turbulence parameterization ------------------------------- $
+$   The LMPN namelist group configures the Langmuir Mixing
+$   Parameterization used within the context of climate modeling and
+$   when WW3 is coupled to an active ocean model (.e.g., POP2, MOM6).
+$   The LMPENABLED flag turns on the parameterization computations.
+$   The SDTAIL flag enables the spectral tail contribution. HSLMODE
+$   parameter determines the surface layer depth (HSL).
+$
+$   Input parameters  : Namelist LMPN
+$                       LMPENABLED : Logical controlling Langmuir
+$                                    related computations.
+$                                     T : Activates calculations
+$                                     F : Calculations off (default)
+$                       SDTAIL     : Logical controlling addition of
+$                                    tail contribution.
+$                                     T : Tail added to Stokes
+$                                         drift (LMPENABLED=T also).
+$                                     F : No tail added (default)
+$                       HSLMODE    : Integer indicating surface layer
+$                                    depth (HSL) type.
+$                                     0 : Uniform HSL=10m (default)
+$                                     1 : HSL from coupler
+$
+$   Output parameters :
```

Comment:
`USS` and `USSH` are not namelists, only output variables.  They are associated with the `LMPN` namelist in that they won't be available unless `LMPN` is turned on.  I listed both the input parameters and output parameters associated with `LMPN`.  I can remove `USS` and `USSH` if it's confusing or doesn't belong.  Please let me know and I can add it to the small fix PR I'll make.

Reply:
Surface Stokes drift is available without LMPN namelist. It's my understanding that SDTAIL=T adds a tail contribution to USS/USSH.   I find it confusing to add output parameters within the namelist description section and therefore recommend removing this here. The manual would be a good spot for any additional information that needs to be added potentially.

Reply:
Okay, I'll remove the output parameters from the namelist section to avoid confusion. 

I had been under the impression that the calculations for USS/USSH are turned on when `LMPENABLED` is set to `T`.  Looking back over it, I think think I misunderstood, and it is just the case for USSH.